### ISSUE TITLE
motif

### ISSUE THREAD

**Issue by hfrisfirsch** [ubccr/software-layer#317]
Hi, I am requesting that motif be installed for the CCR. An easybuild recipe can be found here
https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/motif

thank you!

**Comment by dsajdak** [comment]
@hfrisfirsch I have a test build of this software but I need a way to test it to make sure it's working before it gets published to our software repository.  Would you please email ccr-help@buffalo.edu with an example of how you'll use it and if input files are needed, include a small example file to test?  Thanks!

**Comment by dsajdak** [comment]
This has been published to the ccrsoft/2023.01 software repository

```
module spider motif

---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  motif: motif/2.3.8
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
    Description:
      Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems. It was the standard
      toolkit for the Common Desktop Environment and thus for Unix.


    You will need to load all module(s) on any one of the lines below before the "motif/2.3.8" module is available to load.

      gcccore/11.2.0

    Help:

      Description
      ===========
      Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building
       applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.
       It was the standard toolkit for the Common Desktop Environment and thus for Unix.


      More information
      ================
       - Homepage: https://motif.ics.com/
```
